roofit/roofitcore/inc/RooGlobalFunc.h

```diff
@@ -65,9 +65,13 @@ enum MsgTopic { Generation=1, Minimization=2, Plotting=4, Fitting=8, Integration
 enum MPSplit { BulkPartition=0, Interleave=1, SimComponents=2, Hybrid=3 } ;
 
 /// For setting the batch mode flag with the BatchMode() command argument to
-/// RooAbsPdf::fitTo();
+/// RooAbsPdf::fitTo()
 enum class BatchModeOption { Off, Cpu, Cuda, Old };
 
+/// For setting the offset mode with the Offset() command argument to
```

Comment:
I don't like the name Offset(Template), it seems to me it works only for template fits where, in my understanding, is when the pdf is binned. On the other hand this is valid for all binned likelihood fits.
I would prefer calling it `Offset(bin)`

Reply:
Okay, agreed!